diff --git a/frontend/settings/dev.py b/frontend/settings/dev.py
index 018f1aa..afcbf99 100644
--- a/frontend/settings/dev.py
+++ b/frontend/settings/dev.py
@@ -59,6 +59,8 @@ BROKER_TRANSPORT_OPTIONS = {}
 BROKER_URL = 'mongodb://localhost/celery'
 ########## END CELERY CONFIGURATION
 
+CELERY_REDIRECT_STDOUTS = False
+
 
 ########## TOOLBAR CONFIGURATION
 # See: https://github.com/django-debug-toolbar/django-debug-toolbar#installation
